extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/src/main/java/org/eclipse/smarthome/binding/fsinternetradio/internal/radio/FrontierSiliconRadio.java

```diff
@@ -132,8 +132,7 @@ public void setVolumeAbsolute(int volume) throws IOException {
      * @throws IOException if communication with the radio failed, e.g. because the device is not reachable.
      */
     public void setVolumePercent(int volume) throws IOException {
-        final int newVolumePercent = volume < 0 ? 0 : volume > 100 ? 100 : volume;
-        final int newVolumeAbsolute = (newVolumePercent * 32) / 100;
+        final int newVolumeAbsolute = (volume * 32) / 100;
```

Comment:
I removed this check from here because the PercentType itself does it. In the `handleCommand `method we check if the given command is a PercentType, so we are sure that it is a value in the range [0-100] before getting in the `setVolumePercent `method